apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -935,8 +919,10 @@ describe('[Users]', function () {
 					expect(res.body).to.have.property('count');
 					expect(res.body).to.have.property('total');
 					expect(res.body).to.have.property('users');
-					const lastUser = res.body.users[res.body.users.length - 1];
-					expect(lastUser).to.have.property('active', false);
+					const firstUser = res.body.users.find((u) => u._id === deactivatedUser._id);
+					if (firstUser) {
```

Comment:
firstUser should be always defined here right? I assume it should fail the test if not

(I know this check is redundant, but this could give a few false positives)